### PR TITLE
v4 launch details

### PR DESCRIPTION
**Pull request by mcollina**
Signed-off-by: Matteo Collina <hello@matteocollina.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,60 +68,42 @@ without any changes, if you are just using one of the main classes and
...
-    })
-  ],
-  resolve: {
-    fallback: {
-      buffer: require.resolve('buffer/')
-    }
-  }
-}
-```
-
```
Comment (vweevers) [inline README.md:90]: If this is no longer necessary (how?) then what is the purpose of https://github.com/nodejs/readable-stream/blob/main/test/browser/fixtures/webpack.browser.config.mjs?

Comment (mcollina) [reply README.md:90]: I made some mess with the readme, this is still needed, good spot.

**Comment by benjamingr** [comment]
(with the readme example fix)

File: README.md [hunk 2]
```diff
@@ -20,6 +20,20 @@ Node you, or the users of your libraries are using, use **readable-stream** _onl
 
 As of version 2.0.0 **readable-stream** uses semantic versioning.
 
+## Version 4.x.x
+
+v4.x.x of `readable-stream` is a cut from Node 18. This version supports Node 12, 14, 16 and 18, as well as evergreen browsers.
+The breaking changes introduced by v4 are composed by the combined breaking changes in
```
Comment (vweevers) [inline README.md:26]:
```suggestion
The breaking changes introduced by v4 are composed of the combined breaking changes in:
```

File: README.md [hunk 3]
```diff
@@ -20,6 +20,20 @@ Node you, or the users of your libraries are using, use **readable-stream** _onl
...
+* [Node v12](https://nodejs.org/en/blog/release/v12.0.0/)
+* [Node v13](https://nodejs.org/en/blog/release/v13.0.0/)
+* [Node v14](https://nodejs.org/en/blog/release/v14.0.0/)
+* [Node v15](https://nodejs.org/en/blog/release/v15.0.0/)
+* [Node v16](https://nodejs.org/en/blog/release/v16.0.0/)
+* [Node v17](https://nodejs.org/en/blog/release/v17.0.0/)
+* [Node v18](https://nodejs.org/en/blog/release/v18.0.0/)
+
+This also include _many_ new features.
```
Comment (vweevers) [inline README.md:35]:
```suggestion
This also includes _many_ new features.
```

**Review by vweevers — APPROVED** [review]
The closing of various GH issues after #471 created ambiguity about what is and isn't supported (mainly: does a bundler need to inject shims) and the list of changes is long. Makes me wish we had a complete and standardized changelog. But that's a lot of work and v4 already has been. So LGTM (with two nits) though I do worry about maintainability going forward.

**Pull request merged**